src/constants/libraries.ts

```diff
@@ -0,0 +1,7 @@
+export const LANGUAGES = {
```

Comment:
curious, why constant instead of enum? i believe enums are more suitable for this data type

Reply:
Well, in the Filter component I need all the languages. The solution I tried with enum seemed like a hack, so I made an object in constants instead. Also, extracting a `LanguageType` type from this object.

Reply:
> Well, in the Filter component I need all the languages.

I think you can just use `Object.Keys/Values/Entries`?
https://javascript.plainenglish.io/iterating-on-a-typescript-string-enum-45f7e36b015b

> Also, extracting a LanguageType type from this object.

If it is an `enum`, you would not need to use `keyof typeof`, you can just straight use the `enum` as the type. This is one advantage of using enum over constant